source/features/last-notification-page-button.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '../feature-manager';
+import looseParseInt from '../helpers/loose-parse-int';
+import {assertNodeContent} from '../helpers/dom-utils';
+
+function init(): void | false {
+	const nextButton = select('a[data-hotkey="ArrowRight"]');
```

Comment:
This attribute is added by us https://github.com/refined-github/refined-github/blob/4939c6a2862f9ddcb138874efc627e9b1b8d97ff/source/features/pagination-hotkey.tsx#L24